builtin/providers/aws/resource_aws_db_subnet_group_test.go

```diff
@@ -149,7 +149,7 @@ resource "aws_subnet" "bar" {
 }
 
 resource "aws_db_subnet_group" "foo" {
-	name = "FOO"
+	name = "foo"
```

Comment:
nitpick... shouldn't this be OK to leave as `FOO`? The `StateFunc` should downcast it, so really leaving it as caps or mixed case would test that we're actually doing this.